### DISCUSSION TITLE
Cold/Warm/Oclusion Front spawner

### DISCUSSION THREAD

**Discussion by mesochaser2425** [niels747/2D-Weather-Sandbox#39] [Ideas]
So i dont know if only i have this problem where i am spawning (manualy) a cold front it is pretty messy and whole map gets alot of unnecessary clouds that prevents Cb clouds to form. Front spawner would be really helpful becouse you would have alot more control over the fronts. I dont know maybe im doing something wrong when im spawing the cold fronts but yeah. Maybe some help? thanks!

**Comment by Meso22** [comment]
Sometimes you really got to be careful with the brush intensity usually setting it to the lowest level helps to cool things down much more slowly also turn iterations/frame to one and turn the auto adjust off so that things are even more slow when editing.

**Reply by niels747** [reply]
Good idea in principle, but i'm not sure how to implement something like that...
What helps to create realistic fronts is a really wide simulation, but you need to have a fast GPU to run it at reasonable speed.

**Reply by mesochaser2425** [reply]
i have a pretty good gpu, a Radeon XFX 580 and its running realy smooth in this simulator

**Reply by kimswx** [reply]
did you use temperature tool? because when i use it, it just heats up and doesnt cool down

**Reply by Meso22** [reply]
Hold the control button on your keyboard whilst using the temperature tool to cool things down.

**Reply by kimswx** [reply]
oh, thank you for the info
